doc/release_notes/release_notes_1.2.rst

```diff
@@ -0,0 +1,352 @@
+.. _release_notes_1.2:
+
+ACRN v1.2 (Aug 2019)
+####################
+
+We are pleased to announce the release of ACRN version 1.2.
+
+ACRN is a flexible, lightweight reference hypervisor, built with
+real-time and safety-criticality in mind, optimized to streamline embedded
+development through an open source platform. Check out the :ref:`introduction` for more information.
+All project ACRN source code is maintained in the https://github.com/projectacrn/acrn-hypervisor
+repository and includes folders for the ACRN hypervisor, the ACRN device
+model, tools, and documentation. You can either download this source code as
+a zip or tar.gz file (see the `ACRN v1.2 GitHub release page
+<https://github.com/projectacrn/acrn-hypervisor/releases/tag/v1.2>`_)
+or use Git clone and checkout commands::
+
+   git clone https://github.com/projectacrn/acrn-hypervisor
+   cd acrn-hypervisor
+   git checkout v1.2
+
+The project's online technical documentation is also tagged to correspond
+with a specific release: generated v1.2 documents can be found at https://projectacrn.github.io/1.2/.
+
+Version 1.2 major features
+**************************
+
+What's New in v1.2
+==================
+* OVMF becomes the only virtual boot loader for Service VM to launch Clearlinux,
+  VxWorks or Windows. Secure boot is supported
+* Support Kata container
+* WaaG: USB host (xHCI) mediator
+* Virtualization supports Always Running Timer (ART)
+* Various bugfixes and enhancements
+
+Document updates
+================
+We have many `reference documents available <https://projectacrn.github.io>`_, including:
+
+* :ref:`Using Windows as User VM <using_windows_as_uos>`
+* :ref:`How to sign binaries of the Clear Linux image <sign_clear_linux_image>`
+* :ref:`Using Celadon as User VM <using_celadon_as_uos>`
+* :ref:`SGX Virtualization <sgx_virt>`
+
+We also updated the following documents based on the **Scenario** in this release, including:
```

Comment:
Since I found something else, I noticed this sentence could be improved too:
```
We also updated the following documents based on the newly
defined **Usage Scenarios** in this release, including:
```